rpc/openapi/openapi.yaml

```
@@ -808,12 +808,43 @@ paths:
                 $ref: "#/components/schemas/ErrorResponse"
   /genesis:
     get:
-      summary: Get Genesis
+      summary: Get Genesis in chunked
```

```suggestion
      summary: Get Genesis
```

/genesis and /genesis_chunked are two different routes. 

https://github.com/tendermint/tendermint/blob/master/rpc/core/routes.go#L24

---

Eep. Sorry about this!

---

Eep. Sorry about this!